bento/src/main/java/com/yelp/android/bento/core/ComponentViewHolder.kt

```diff
@@ -16,6 +16,12 @@ import android.view.ViewGroup
  */
 abstract class ComponentViewHolder<P, T> {
 
+    /**
+     * Contains the absolute position within the entire [ComponentController]. This should only be
+     * set by the [ComponentController].
+     */
+    var absolutePosition: Int = -1
```

Comment:
The problem is that doesn't compile. I think it is because the RecyclerViewComponentController is in java, which doesn't recognize internal, and it is probably package protected then.